sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallConnectionAsync.java

```diff
@@ -162,30 +162,6 @@ public String getCallConnectionId() {
         return playAudioWithResponseInternal(audioFileUri, playAudioOptions, null);
     }
 
-    Mono<Response<PlayAudioResult>> playAudioWithResponseInternal(
```

Comment:
Why was this removed?

Reply:
This was not being used, so removed. 

To give a bit background, we used to have 4 version of playAudio() service methods:
PlayAudioResult playAudio(String audioFileUri, boolean loop, String audioFileId, String callbackUri, String perationContext);
Response<PlayAudioResult> playAudio(String audioFileUri, boolean loop, String audioFileId, String callbackUri, String perationContext, final Context context);
PlayAudioResult playAudio(String audioFileUri, PlayAudioOptions playAudioOptions);
Response<PlayAudioResult> playAudioWithResponse(String audioFileUri, PlayAudioOptions playAudioOptions,
        final Context context);
        
Based on one of the feedbacks on ApiView from Srikanta that we need only one method with Response, we removed the second one, thus this internal one became unused.